roles/openshift_master/tasks/main.yml

```diff
@@ -27,6 +27,9 @@
   action: "{{ ansible_pkg_mgr }} name={{ openshift.common.service_type }}-master{{ openshift_version | default('') | oo_image_tag_to_rpm_version(include_dash=True) }} state=present"
   when: not openshift.common.is_containerized | bool
 
+- name: Install bash completion for oc tools
+  action: "{{ ansible_pkg_mgr }} name=bash-completion state=present"
```

Comment:
Sorry for the hastey merge, i pondered master versus openshift_cli but decided master made sense as that's where the kubeconfig was. I missed the containerized check though.